test/quantization/test_numeric_suite_fx.py

```diff
@@ -142,3 +144,90 @@ def compare_and_validate_results(float_model, q_model):
         q_model = convert_fx(prepared_model)
 
         compare_and_validate_results(backup_prepared_model, q_model)
+
+    @skipIfNoFBGEMM
+    def test_compare_model_stub_conv_static_fx(self):
+        r"""Compare the output of static quantized conv layer and its float shadow module"""
+
+        def calibrate(model, calib_data):
+            model.eval()
+            with torch.no_grad():
+                for inp in calib_data:
+                    model(*inp)
+
+        def compare_and_validate_results(float_model, q_model, module_swap_list, data):
+            ob_dict = compare_model_stub_fx(
+                float_model, q_model, module_swap_list, data
+            )
+            self.assertEqual(len(ob_dict), 1)
+            for k, v in ob_dict.items():
```

Comment:
Would be good to add the check ob_dict.keys() are as expected.

Reply:
Done